packages/react-router/lib/server-runtime/data.ts

```diff
@@ -20,61 +20,18 @@ export interface AppLoadContext {
  */
 export type AppData = unknown;
 
-export async function callRouteAction({
```

Comment:
Once the undefined check was removed there was no difference left between `callRouteLoader` and `callRouteAction` so they are now collapsed to `callRouteHandler`